flake.nix

```
@@ -24,211 +25,299 @@
-            inherit ((import flake-compat {
-          pkgs = final;
-          inherit (final.cardanoDbSyncProject.hsPkgs.cardano-node) src;
-        }).defaultNix.packages.${final.system}) cardano-node cardano-cli;
-            inherit (final.cardanoDbSyncProject.exes) cardano-db-sync cardano-smash-server cardano-db-tool;
-          };
-        nixosModules = {
-          cardano-db-sync = { pkgs, lib, ... }: {
-            imports = [ ./nix/nixos/cardano-db-sync-service.nix ];
-            services.cardano-db-sync.dbSyncPkgs =
-              lib.mkDefault self.legacyPackages.${pkgs.system};
-          };
-        };
-      };
+                    # Copy exes to intermediate dir
+                    cp \
+                      --no-clobber \
+                      --remove-destination \
+                      --verbose \
+                      ${lib.concatMapStringsSep " " (exe: "${exe}/bin/*") exes} \
+                       ./
+
+                    # Rewrite libs on macos (from iohk-utils)
+                    ${lib.optionalString
+                       (platform == "macos")
+                       (lib.concatMapStrings
+                         (exe: "rewrite-libs . ${exe}/bin/*")
+                         exes)}
+
+                    # Package distribution
+                    dist_file=${name}.tar.gz
+                    tar -czf $out/$dist_file .
+
+                    # Write summary file
+                    mkdir $out/nix-support
+                    echo "file binary-dist $out/$dist_file" > $out/nix-support/hydra-build-products
+                  '';
+
+            cardano-db-sync-linux = mkDist "linux" project.projectCross.musl64;
+            cardano-db-sync-macos = mkDist "macos" project;
+            cardano-db-sync-docker = callPackage ./nix/docker.nix {
+              inherit (inputs.iohkNix.lib) evalService;
+            };
+
+          in rec {
+            hydraJobs = callPackages inputs.iohkNix.utils.ciJobsAggregates {
+              ciJobs = flake.hydraJobs;
+            } // lib.optionalAttrs (system == "x86_64-linux") {
+              inherit cardano-db-sync-linux cardano-db-sync-docker;
+            } // lib.optionalAttrs (system == "x86_64-darwin") {
+              inherit cardano-db-sync-macos;
+            } // {
+              checks = staticChecks;
+            };
+
+            checks = staticChecks;
+
+            packages = lib.optionalAttrs (system == "x86_64-linux") {
+              inherit cardano-db-sync-linux cardano-db-sync-docker;
+
+              default = flake.packages."cardano-db-sync:exe:cardano-db-sync";
+            } // lib.optionalAttrs (system == "x86_64-darwin") {
+              inherit cardano-db-sync-macos;
+            };
+
+            devShells = with inputs.devxShell.devShells."${system}"; {
+              default = ghc8107-iog;
+              ghc8107 = flake.devShells.default; # Restore previous default
+              devx = ghc8107-iog;
+              devx-ghc962 = ghc962-iog;
```

```suggestion
              devx-ghc96 = ghc96-iog;
```
I think this will work, there are aliases now for the latest minor version to each major version.